authors/apps/authentication/tests/test_reset_password.py

```diff
@@ -0,0 +1,85 @@
+import json
+
+from django.test import TestCase
+from rest_framework.test import APIClient
+from rest_framework import status
+from django.core import mail
+
+
+class PasswordTestCase(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.forgot_password = {
+            'email': 'remmy@test.com'
+        }
+
+        self.reset_password = {
+            'email': 'remmy@test.com',
+            'token': '1234567890',
+            'new_password': 'Password123'
+        }
+
+        self.renew_password = {
+            'email': 'remmy@test.com',
+            'old': 'password1',
+            'new_password': 'Password123'
+        }
+
+        self.old_password = {
+            'email': 'remmy@test.com',
+            'old': 'password1'
+        }
+
+    def test_forgot_password(self):
```

Comment:
This function name and the docsting are quite conflicting. Forgot password and checking if a user receives reset password are a bit different.

Reply:
I have changed the function name and added a more descriptive docstring